directus-solidstart/index.md

```diff
@@ -0,0 +1,314 @@
+---
+title: "Getting Started with Directus and SolidStart"
+description: "Let's learn how to build a blog using Directus as a CMS. You will store, retrieve, and use global metadata such as the site title, create new pages dynamically based on Directus items"
+author:
+  name: "Atila Fassina"
+  avatar_file_name: "author.webp"
+---
+
+## Summary
+
+[SolidStart](https://start.solidjs.com) a meta-framework built around [SolidJS](https://solidjs.com) - SolidStart is a framework aiming to allow people and teams to built high-end applications while bringing their own opinions. In this tutorial, you will learn how to build a website using Directus as a CMS. You will store, retrieve, and use global metadata such as the site title, create new pages dynamically based on Directus items, and build a blog.
+
+## Before You Start
+
+You will need:
+
+- To install Node.js and a code editor on your computer.
+- A Directus project - follow our quickstart guide if you don't already have one.
+- SolidJS knowledge will definitely help, but will not prevent you from following this guide.
```

Comment:
```suggestion
- Node.js and a code editor.
- A Directus project - [follow our quickstart guide](https://docs.directus.io/getting-started/quickstart) if you don't already have one.
- (Bonus) SolidJS knowledge will definitely help, but is not required.
```

This suggestion is mostly an act of being more direct, removing words that don't add value, and linking users to the quickstart which they may need.